guides/get_started/installation.md

```diff
@@ -0,0 +1,262 @@
+  use Backpex.LiveResource,
+    layout: {MyAppWeb.Layouts, :admin},
+    schema: MyApp.Blog.Post,
+    repo: MyApp.Repo,
+    update_changeset: &MyApp.Blog.Post.update_changeset/3,
+    create_changeset: &MyApp.Blog.Post.create_changeset/3,
+    pubsub: MyApp.PubSub,
+    topic: "posts",
+    event_prefix: "post_",
+end
+```
+
+`Backpex.LiveResource` is the module that will generate the corresponding LiveViews for the resource you configured. We provide a macro you have to use to configure the LiveResource. You are required to set some general options to tell Backpex where to find the resource and what changesets should be used. The above example shows the configuration for a `Post` resource.
+
+All options you can see in the above example are required:
+
+- The `layout` option tells Backpex which layout to use for the LiveResource. In this case, we use the `:admin`(`admin.html.heex`) layout created in the previous step.
+- The `schema` option tells Backpex which schema to use for the resource.
+- The `repo` option tells Backpex which repo to use for the resource.
+- The `update_changeset` and `create_changeset` options tell Backpex which changesets to use for updating and creating the resource.
+- The `pubsub` option tells Backpex which pubsub to use for the resource (see the [Listen to PubSub Events](live_resource/listen-to-pubsub-events.md) guide for more information).
+- The `topic` option tells Backpex which topic to use for the resource when broadcasting events.
+- The `event_prefix` option tells Backpex which event prefix to use for the resource when broadcasting events.
+
+In addition to the required options, you pass to the `Backpex.LiveResource` macro, you are required to implement the following callback functions in the module:
+
+- [`singular_name/0`](Backpex.LiveResource.html#c:singular_name/0) - This function should return the singular name of the resource.
+- [`plural_name/0`](Backpex.LiveResource.html#c:plural_name/0) - This function should return the plural name of the resource.
+- [`fields/0`](Backpex.LiveResource.html#c:fields/0) - This function should return a list of fields to display in the LiveResource.
+
+After implementing the required callback functions, our `PostLive` module looks like this:
+
+```elixir
+defmodule MyAppWeb.Live.PostLive do
+  use Backpex.LiveResource,
+    layout: {MyAppWeb.Layouts, :admin},
+    schema: MyApp.Blog.Post,
+    repo: MyApp.Repo,
+    update_changeset: &MyApp.Blog.Post.update_changeset/3,
+    create_changeset: &MyApp.Blog.Post.create_changeset/3,
+    pubsub: MyApp.PubSub,
+    topic: "posts",
+    event_prefix: "post_",
+
+  @impl Backpex.LiveResource
+  def singular_name, do: "Post"
+
+  @impl Backpex.LiveResource
+  def plural_name, do: "Posts"
+
+  @impl Backpex.LiveResource
+  def fields do
+  [
+    title: %{
+      module: Backpex.Fields.Text,
+      label: "Title"
+    },
+    views: %{
+      module: Backpex.Fields.Number,
+      label: "First Name"
```

Comment:
Maybe you meant Likes or something else?

Reply:
yes, probably a copy & paste mistake. see https://github.com/naymspace/backpex/pull/342